### ISSUE TITLE
Can't import bertopic in pycharm but runs fine in Anaconda prompt console

### ISSUE THREAD

**Issue by Garren87** [MaartenGr/BERTopic#2062]
### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Desribe the bug

Hello, I am using python 3.11 and bertopic 0.16.2 in Pycharm, and I have only got CPU i9-10900K for training.
Seems that I am facing similar problem with #776 , the import code in Pycharm keeps running for more that one hour and never finish.
 ```python
from bertopic import BERTopic

```
However, I am able to work in the Anaconda prompt, actually I have finished the example of Dynamic Topic model in [https://maartengr.github.io/BERTopic/getting_started/topicsovertime/topicsovertime.html](url)
I have tried to some steps like reinstall bertopic and create environment, but none of them fixing the problem.
Thank you for your time!

### Reproduction

```python
from bertopic import BERTopic

```

### BERTopic Version

0.16.2

**Comment by MaartenGr** [comment]
I'm using VSCode rather than Pycharm so haven't run into this issue before. Are you sure you are using the same conda environment in Pycharm?

**Comment by Garren87** [comment]
Thank you for your kind reply, I have found that this problem can be solved by using conda environment of python 3.8 (instead of 3.7 or 3.11) in Pycharm.

**Comment by MaartenGr** [comment]
Glad to hear that the issue was resolved and thank you for posting the solution!
